cms/djangoapps/api/v1/tests/test_serializers/test_course_runs.py

```diff
@@ -16,39 +16,45 @@
 @ddt.ddt
 class CourseRunSerializerTests(ModuleStoreTestCase):
 
-    @ddt.data(
-        ('instructor_paced', False),
-        ('self_paced', True),
-    )
-    @ddt.unpack
-    def test_data(self, expected_pacing_type, self_paced):
-        start = datetime.datetime.now(pytz.UTC)
-        end = start + datetime.timedelta(days=30)
-        enrollment_start = start - datetime.timedelta(days=7)
-        enrollment_end = end - datetime.timedelta(days=14)
-
-        course = CourseFactory(
-            start=start,
-            end=end,
-            enrollment_start=enrollment_start,
-            enrollment_end=enrollment_end,
-            self_paced=self_paced
-        )
+    def setUp(self):
+        super(CourseRunSerializerTests, self).setUp()
+
+        self.start = datetime.datetime.now(pytz.UTC)
+        self.end = self.start + datetime.timedelta(days=30)
+
+        self.request = RequestFactory().get('')
+
+    def get_course(self, self_paced, enrollment_start=None, enrollment_end=None):
+
+        if enrollment_start and enrollment_end:
```

Comment:
What if we do not check `enrollment_start` and `enrollment_end` and pass `None` to `CourseFactory` will it break? 

I see the following both are equlant 
```
course1 =  CourseFactory(start=self.start, end=self.end, self_paced=self_paced)

course2 = CourseFactory(start=self.start, end=self.end, self_paced=self_paced, enrollment_start=None, enrollment_end=None)

```
Does course1.enrollment_start and course2.enrollment_start are both None? Can you check?

Reply:
No, it will not break. 
`course1` and `course2` are not equivalent. This is because `course1` doesn't have  enrollment_start and enrollment_end attributes.